### DISCUSSION TITLE
Implementing passwordless auth

### DISCUSSION THREAD

**Discussion by omar2205** [robingenz/capacitor-firebase-authentication#116] [Ideas]
🤔 How would I get passwordless auth to work?

**Comment by robingenz** [comment]
Do you mean [Firebase Email Link Authentication](https://firebase.google.com/docs/auth/web/email-link-auth)?

**Comment by omar2205** [comment]
Yes, I'm trying to think of a clean way to use that with a capacitor app.

**Reply by robingenz** [reply]
Okay, that's currently not supported by this plugin. I've created a feature request to track progress: #117.
